searx/templates/simple/preferences.html

```diff
@@ -298,18 +298,18 @@ <h1>{{ _('Preferences') }}</h1>
     <p>{{ _('Currently used search engines') }}</p>
     {{ tabs_open() }}
      {% set ns = namespace(checked=true) %}
-     {% for categ in categories_as_tabs + [OTHER_CATEGORY] %}
+     {% for categ in categories_as_tabs + [DEFAULT_CATEGORY] %}
      {{ tab_header('enginetab', 'category' + categ, _(categ), ns.checked )}}
      {% set ns.checked = false %}
-   {% if categ == OTHER_CATEGORY %}
-      <p>{{_('This tab does not show up for search results, but you can search the engines listed here via bangs.')}}</p>
-    {% endif %}
+     {% if categ == DEFAULT_CATEGORY %}
+      <p>{{_('This tab dues not exists in the user interface, but you can search in these engines by its !bangs.')}} <a href="{{ url_for('info', pagename='search-syntax') }}">&#9432;</a></p>
+     {% endif %}
     <div class="scrollx">
     <table class="striped table_engines">
       <tr>{{- "" -}}
         <th class="engine_checkbox">{{ _("Allow") }}</th>{{- "" -}}
         <th class="name">{{ _("Engine name") }}</th>{{- "" -}}
-        <th class="shortcut">{{ _("Shortcut") }}</th>{{- "" -}}
+        <th class="shortcut">{{ _("!bang") }}</th>{{- "" -}}
```

Comment:
Does that mean the engine name is not a bang? What would be the name?

Should this section be adjusted?

https://github.com/searxng/searxng/blob/393e14965acbf0c9a1942a249a55705be9881918/searx/infopage/en/search-syntax.md?plain=1#L7-L26

Reply:
> Does that mean the engine name is not a bang? What would be the name?

No, I haven't thought that way when I changed from "shortcut" to "!bang" in the column header.

> Should this section be adjusted?

I don't think so, the engine name is still a valid !bang .. 

My thought process was: I think it's hard to show all possibilities in the user interface (preferences) without overloading the UI ... the users will usually use the *shortcut bang*.  On the info page  can read that the *engine bang* is also a valid search syntax.

> Or should the !bang column display, for example, `!archive_is`, `!ai` for the archive.is engine?

Yes this is what I intend, but in a follow up PR.  Now, when I read your comment, think it was not a good idea of mine to add the change from "shortcut" to "!bang" in this PR -- > its not related to this PR / sorry.

Reply:
> Or should the !bang column display, for example, `!archive_is`, `!ai` for the archive.is engine?

In https://github.com/searxng/searxng/pull/2327 the !bang from shortcuts, categories and the engine name are displayed in the tool-tip.